prac_06/used_cars.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404/CP5632 Practical - Client code to use the Car class.
+Note that the import has a folder (module) in it.
+This is why we name our folders with no spaces or capitals, as valid module names.
+"""
+
+from prac_06.car import Car
+
+
+def main():
+    """Demo test code to show how to use car class."""
+    car1 = Car(100, "Limo")
```

Comment:
(Optional) You could make the "car1" name match "Limo", so that it doesn't get confusing later on in testing.